packages/dd-trace/test/appsec/iast/analyzers/nosql-injection-mongodb-analyzer.mongoose.plugin.spec.js

```diff
@@ -27,11 +27,16 @@ describe('nosql injection detection in mongodb - whole feature', () => {
         const dbName = id().toString()
         mongoose = require(`../../../../../../versions/mongoose@${mongooseVersion}`).get()
 
-        mongoose.connect(`mongodb://localhost:27017/${dbName}`, {
+        await mongoose.connect(`mongodb://localhost:27017/${dbName}`, {
           useNewUrlParser: true,
           useUnifiedTopology: true
         })
 
+        if (mongoose.models.Test) {
+          delete mongoose.models?.Test
```

Comment:
why?

Reply:
We were running this test only with express v4.21.1. Now, it runs with all express versions, and this part was necessary to clean the Test model before creating a new one

Reply:
oh, got it, as we are requiring same mongoose version for each express version, require is caching it. 👍